server-essentials

```diff
@@ -135,6 +136,14 @@ task "Cleaning up"; clean; taskdone
 
 
 
+echo ""
+echo "$(tput bold)Welcome to server-essentials!"
+echo "Written by Rayzr522 & (a little but) ACA30"
```

Comment:
Please move this and the "if you want a plugin added, create an issue" to before all the installation of the plugins and the setup of the server.